code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -630,6 +630,16 @@
 
 /datum/reagent/gluttonytoxin/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
 	L.ForceContractDisease(new /datum/disease/transformation/morph(), FALSE, TRUE)
+	
+/datum/reagent/pacmanimfunnyrightilikepacmanthatmakesmefunnyplease
```

Comment:
pacmanimfunnyrightilikepacmanthatmakesmefunnyplease change this

Reply:
ok

Reply:
lmao what the fuck